### PR TITLE
Fix spelling, spacing, and unused variables

### PR DESCRIPTION
**Pull request by ryan1288**
Minor PR fixing spelling mistakes, spacing, and removed unused variables. Issue partially mentioned in https://github.com/opencv/opencv_zoo/issues/135#issuecomment-1962323969.

### CODE REVIEW

**Comment by ryan1288** [comment]
Additional note: While I did not implement this change, I observed that the following was declared but unused in the C++ demos.
```
void setBackendAndTarget(dnn::Backend bId, dnn::Target tId)
{
    this->backendId = bId;
    this->targetId = tId;
    this->net.setPreferableBackend(this->backendId);
    this->net.setPreferableTarget(this->targetId);
}
```

File: models/person_detection_mediapipe/demo.cpp [hunk 1]
```diff
@@ -237,10 +237,9 @@ int main(int argc, char** argv)
         backendTargetPairs[backendTargetid].first, backendTargetPairs[backendTargetid].second);
     //! [Open a video file or an image file or a camera stream]
     if (!cap.isOpened())
-        CV_Error(Error::StsError, "Cannot opend video or file");
+        CV_Error(Error::StsError, "Cannot  video or file");
```
Comment (fengyuentau) [inline models/person_detection_mediapipe/demo.cpp:240]: open

**Comment by fengyuentau** [comment]
> Additional note: While I did not implement this change, I observed that the following was declared but unused in the C++ demos.
> 
> ```
> void setBackendAndTarget(dnn::Backend bId, dnn::Target tId)
> {
>     this->backendId = bId;
>     this->targetId = tId;
>     this->net.setPreferableBackend(this->backendId);
>     this->net.setPreferableTarget(this->targetId);
> }
> ```

Which particular demo?

File: models/person_detection_mediapipe/demo.cpp (hunk 1 above)
Comment (ryan1288) [reply models/person_detection_mediapipe/demo.cpp]: Fixed! Also updated several more typos.

**Comment by ryan1288** [comment]
Models with that unused function:

- `face_detection_yunet/demo.cpp`
- `object_detection_yolox/demo.cpp`
- `person_detection_mediapipe/demo.cpp`
- 2x in `pose_estimation_mediapipe/demo.cpp` for each class

In the Python equivalent classes and `benchmark.py`, it is used for benchmarking. However, currently, they are unused in the C++ versions. Do we plan to keep them for future evaluation?

**Comment by fengyuentau** [comment]
> In the Python equivalent classes and benchmark.py, it is used for benchmarking. However, currently, they are unused in the C++ versions. Do we plan to keep them for future evaluation?

Well, you are right. They are used for benchmark in Python scripts but we don't need them in C++ demos. It's okay to remove them then.

**Comment by ryan1288** [comment]
@fengyuentau  I've removed all such instances of unused `setBackendAndTarget` from C++ demos. I also reran all the C++ demos to ensure functionality was not affected.

**Review by fengyuentau — APPROVED** [review]

**Comment by fengyuentau** [comment]
Thank you for your contritbution 👍

**Pull request merged**